src/cloud/docker/docker-containers-service.md

```diff
@@ -63,6 +62,14 @@ See [Manage the database] for details about using the database.
 
 The Elasticsearch container for {{site.data.var.mcd-prod}} is a standard Elasticsearch container with required plugins and configurations for {{site.data.var.ee}}.
 
+You can customize the Elasticsearch container using the `--es-env-var` option when you generate the Docker Compose configuration file. You can set Elasticsearch options and specify the environment variables to apply when the container starts, such as the heap size for JVM.
+
+```bash
+php vendor/bin/ece-docker build:compose --es-env-var=ES_JAVA_OPTS="-Xms512m -Xmx512m" --es-env-var=node.store.allow_mmapfs=false
```

Comment:
The options here, I assume someone working with ES would know what this means...otherwise, it could make a noob's eyes swirl.

Reply:
I added a link to Elasticsearch configuration setting docs on line 71.